speed-dreams/source-2.2.3/data/data/menu/DataSelectionMenu.xml

```diff
@@ -22,7 +22,8 @@
         <section name="CheckboxCarData">
             <attstr name="type" val="check box"/>
             <attstr name="text" val="Car data"/>
-            <attstr name="tip" val="Enable/Disable setting"/>
+            <attstr name="tip"
+                    val="Save the position, rotation, speed, acceleration, and gear of the car for each timestamp"/>
```

Comment:
Shouldn't this be timestep? 
Timestamp: a digital record of the time of occurrence of a particular event.
Timestep: generally used to mean one step in for example a simulation